googlemaps/places.py

```diff
@@ -0,0 +1,392 @@
+#
+# Copyright 2015 Google Inc. All rights reserved.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+"""Performs requests to the Google Maps Places API."""
+
+from googlemaps import convert
+
+
+def places(client, query, location=None, radius=None, language=None,
+           min_price=None, max_price=None, open_now=False, types=None,
+           page_token=None):
+    """
+    Performs text search for places.
+
+    :param query: The text string on which to search, for example: "restaurant".
+    :type query: string
+
+    :param location: The latitude/longitude value for which you wish to obtain the
+                     closest, human-readable address.
+    :type location: dict or list or tuple
```

Comment:
In some places we did and others we didn't, and now I've unified all the lat/lng handling in c4694adf36a92f37eab84b7450bb6c17a0e7903e